cmd/restic/integration_test.go

```diff
@@ -59,9 +59,11 @@ func newTestErrorChannel() chan error {
 
 func (w *webhookserver) runWebServer(t *testing.T) {
 	mux := http.NewServeMux()
-	mux.HandleFunc("/", func(wr http.ResponseWriter, r *http.Request) {
-		w.jsonData, _ = io.ReadAll(r.Body)
-	})
+	mux.HandleFunc(
```

Comment:
Are you using some custom formatter? Is it possible to revert these formatting changes?

These format changes are a bit irritating and create a ton of changes in the diff. Sorry.

Reply:
I'm sorry, I fix it and change my idea formatter

Reply:
Fixed

Reply:
Thanks!

I'll try to have another in-depth look at the code this week.

Reply:
fixed